config/routes.rb

```diff
@@ -0,0 +1,20 @@
+Rails.application.routes.draw do
+  mount Maestrano::Connector::Rails::Engine, at: '/'
+  mount Sidekiq::Web => '/sidekiq'
```

Comment:
I moved the Sidekiq blok in `initializers/sidekiq.rb` to keep `routes.rb` cleaner.